api/lib/gateways/Academy-Benefits/FetchRecord.js

```diff
@@ -48,25 +46,27 @@ async function fetchCustomer(id, fetchDB, fetchAPI, logger) {
 }
 
   const recordEquality = (record1, record2) => {
-    console.log(record2);
-    console.log(record1);
+    if (record1 === record2) return true;
```

Comment:
Is it worth having this here? If both record variables point to objects, this will never evaluate to true unless they both point to the same object.

Reply:
This is in case they are both null/ undefined. Maybe not the best way of writing it though.

Reply:
Good point, have removed this

Reply:
@elcorbs I think that would be caught by the following line?

Reply:
Ah I see - you mean it should return true if both are null/undefined.

Yes, mildly confusing for sure.

Reply:
(Could do `if (!record1 && !record2) return true` - but not sure that's any less puzzling at first glance!)

Reply:
Yep, that probably reads better. Tbh though I thought removing it would make tests fail, but it doesn't seem to have so maybe all is good